app/models/user.rb

```diff
@@ -4,5 +4,21 @@ class User < ApplicationRecord
   devise :invitable, :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
 
-  validates :full_name, presence: true, uniqueness: true
+  validates :full_name, presence: true
+
+  def admin?
+    role == "admin"
```

Comment:
Admin and RDA are the same. I've just kept it generic to be consistent with advanced and basic